pytest_splunk_addon/splunk.py

```diff
@@ -322,3 +273,60 @@ def splunk_web_uri(splunk):
     uri = f'http://{splunk["host"]}:{splunk["port_web"]}/'
     LOGGER.info("Fetched splunk_web_uri=%s", uri)
     return uri
+
+
+def is_responsive_splunk(splunk):
+    """
+    Verify if Splunk is responsive or not
```

Comment:
this only makes sure that you can talk to splunkd, doesn't guarantee that the UI is up or that mongod is ready as well.

Reply:
Updated the docstrings.